pov.md

```diff
@@ -17,19 +17,18 @@ in the graph, by pulling it up to the root and dragging its relationships along
 with it. So the same graph from 6's perspective would look like:
 
 ```
-          6
-          |
-    7-----+-----2
-                |
-                0
-          1-----+-----3
-          |           |
-        4-+-5       8-+-9
+        6
+        |
+  7-----+-----2
+              |
+        1-----0-----3
+        |           |
+      4-+-5       8-+-9
```

Comment:
I aligned this with the above graph and used the same notation for a group of three children as used in the above graph.